idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -9,8 +9,10 @@ export default `
           <div class="details-header">
             <h1 data-hook="title"></h1>
             <div class="idea-controls">
-              <button data-hook="uninstall" class="button primary">Disable <span data-hook="title"></span></button>
+              <button data-hook="feedback" class="button primary">Give Feedback</button>
+              <button data-hook="uninstall" class="button">Disable <span data-hook="title"></span></button>
               <button data-hook="install" class="button primary">Enable <span data-hook="title"></span></button>
+              <div class="user-count">7,654,321</div>
```

Comment:
This was removed in a previous pr, via postponing user-counts until post-mozlando.